docs/src/guide/extensions_cluster_autoscaler.md

```diff
@@ -13,6 +13,66 @@ The following parameters may be added on each pool definition to enable manageme
 * `min_size`: Define the minimum scale of a pool managed by the cluster autoscaler. Defaults to `size` when not provided.
 * `max_size`: Define the maximum scale of a pool managed by the cluster autoscaler. Defaults to `size` when not provided.
 
+The cluster-autoscaler will manage the size of the nodepools with the attribute `autoscale = true`. To avoid the conflict between the actual `size` of a nodepool and the `size` defined in the terraform configuration files, you can add the `ignore_size = true` attribute to the nodepool definition in the `worker_pools` variable. This parameter will allow terraform to ignore the [drift](https://developer.hashicorp.com/terraform/tutorials/state/resource-drift) of the size parameter for the specific nodepool.
+
+This setting is strongly recommended for nodepools configured with `autoscale = true`.
+
+Example:
+
+```
+worker_pools = {
+  np-autoscaled = {
+    description = "Node pool managed by cluster autoscaler",
+    size        = 2,
+    min_size    = 1,
+    max_size    = 3,
+    autoscale   = true,
+    ignore_size = true # allow nodepool size drift
+  },
+  np-autoscaler = {
+    description      = "Node pool with cluster autoscaler scheduling allowed",
+    size             = 1,
+    allow_autoscaler = true,
```

Comment:
As above, if I don't set the `ignore_size`, will it default to `true` and allow drift or `false` and not allow drift?

Reply:
When it's not set, defaults to `false` and is not allowing configuration drift. (in line with the old behavior).

Reply:
ok, please document this in a separate page for autoscaler.

Reply:
I've update the docs and mentioned the new feature in two places:
- nodepool parameters - because it's a parameter relevant for the nodepools
- cluster autoscaler - because it makes sense to be set when cluster-autoscaler extension is used.